lib/pages/calendar_attendance_page.dart

```diff
@@ -0,0 +1,180 @@
+import 'package:flutter/material.dart';
+import 'package:table_calendar/table_calendar.dart';
+
+class CalendarAttendancePage extends StatefulWidget {
+  @override
+  _CalendarAttendancePageState createState() => _CalendarAttendancePageState();
+}
```

Comment:
Add parameter to accept, focusDate, default is datetime.now(), then line 22 make it widget.focusDate